src/utils.ts

```diff
@@ -55,10 +57,10 @@ export function graphqlRequest(
       query,
       variables,
     }),
-  }).then((responce) => {
-    if (responce.ok) return responce.json();
-    return responce.text().then((body) => {
-      throw Error(`${responce.status} ${responce.statusText}\n${body}`);
+  }).then((response) => {
+    if (response.ok) return response;
```

Comment:
This now returns `response` rather than `response.json()`, because we need to be able to get the `response.headers` later.